package.json

```diff
@@ -5,6 +5,9 @@
   "repository": "https://github.com/FormidableLabs/aws-lambda-serverless-reference",
   "author": "Ryan Roemer <ryan.roemer@formidable.com>",
   "license": "MIT",
+  "engines": {    
+    "node": ">=10"
```

Comment:
(picking up on actual visible code thread) My vote here is remove the `engines` until we use something that is _actually_ node10+ and use `.nvmrc` as our best practice means of getting folks on the right engine. AWS will still support create + update node8 for a few more months and nothing has changed in our very node8 friendly code...

... but I'm not going to block on this if you have consensus from the backend devs team and feel it's a move we should take here.

Reply:
sounds good, i'll remove it